api/swagger.yaml

```diff
@@ -5723,6 +5723,13 @@ paths:
                 description: "Exit code of the container"
                 type: "integer"
                 x-nullable: false
+              Error:
+                description: "container waiting error, if any"
+                type: "object"
+                properties:
+                  Message:
+                    description: "Details of an error"
+                    type: "string"
```

Comment:
Should this use the standard error response type? Similar to https://github.com/kolyshkin/moby/blob/f963500c544daa3c158c0ca3d2985295c875cb6b/api/swagger.yaml#L4334-L4337

Reply:
Hm, guess we can't because of the existing `StatusCode` in the response 😞

Reply:
AFAIR I tried using `ErrorDetail` and I could not, as it creates a circular package dependency :scream: